src/lib/main/main-draft.js

```diff
@@ -0,0 +1,23 @@
+async function pullRequestOpened(context) {
+  context.github.issues.addLabels(
+    context.issue({
+      labels: ["WIP :construction_worker_man:"]
+    })
+  );
+}
+
+async function pullRequestReopened(context) {
+  // nothing to do
+  return context;
+}
+
+async function pullRequestChanged(context) {
+  //nothing to do
+  return context;
```

Comment:
remove `return` and `// nothing to do` line and log warning with text `no pull request changed for draft`

Reply:
What to do about context? I mean should I remove it as a param, because if I don't use it the node will start to complain about the unused var?

Reply:
yes, just delete it, JS won't complain about calling `main-draft.pullRequestChanged(context)`.
Anyway I think it's time to move to a POO approach, we will have to consider to add typescript to the project. But this is for later.